### ISSUE TITLE
Images lack alt text

### ISSUE THREAD

**Issue by lfrancis** [usnationalarchives/nara-bia-photos#106]
https://nara-bia-photos.herokuapp.com

100 images are flagged by axe:

Issue description
Ensures <img> elements have alternate text or a role of none or presentation

Element location
.dLCQfc > img
Element source
`<img srcset="https://catalog.image.drupalme.net/iiif/2/media00%2Fkansas-city%2Frg-075%2F285796%2F75-SR-5039_001.jpg/full/!600,600/0/default.jpg 1x" class="bg-image-source">`

To solve this violation, you need to...
Fix at least one (1) of these issues:
Element does not have an alt attribute

aria-label attribute does not exist or is empty

aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty

Element has no title attribute or the title attribute is empty

Element's default semantics were not overridden with role="presentation"

Element's default semantics were not overridden with role="none"
